samples/sandbox/src/main/kotlin/com/bumble/appyx/sandbox/client/spotlight/SpotlightFaderThrough.kt

```diff
@@ -0,0 +1,89 @@
+package com.bumble.appyx.sandbox.client.spotlight
+
+import android.annotation.SuppressLint
+import androidx.compose.animation.core.FastOutSlowInEasing
+import androidx.compose.animation.core.Transition
+import androidx.compose.animation.core.animateFloat
+import androidx.compose.animation.core.tween
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.draw.scale
+import com.bumble.appyx.core.navigation.transition.ModifierTransitionHandler
+import com.bumble.appyx.core.navigation.transition.TransitionDescriptor
+import com.bumble.appyx.navmodel.spotlight.Spotlight
+
+class SpotlightFaderThrough<T>() : ModifierTransitionHandler<T, Spotlight.State>() {
+
+    @SuppressLint("ModifierFactoryExtensionFunction")
+    override fun createModifier(
+        modifier: Modifier,
+        transition: Transition<Spotlight.State>,
+        descriptor: TransitionDescriptor<T, Spotlight.State>
+    ): Modifier = modifier.composed {
+        val alpha = transition.animateFloat(
+            transitionSpec = {
+                when (targetState) {
+                    Spotlight.State.ACTIVE -> tween(
+                        durationMillis = 210,
```

Comment:
This is a rather random number. Maybe a constant would make sense? I assume it's a combination of 200ms plus 10ms from the `delayMillis`?

Reply:
These timings are taken from material spec for this kind of transition. Overall duration should be 300ms, more details here https://m2.material.io/design/motion/the-motion-system.html#fade-through
<img width="397" alt="image" src="https://user-images.githubusercontent.com/8353068/208858787-5ad032db-2048-4d45-987d-bc4521feb563.png">

Reply:
10 ms delay was a mistake though, thanks for highlighting